spring-cloud-dataflow-admin-starter/src/main/java/org/springframework/cloud/dataflow/admin/config/AdminConfiguration.java

```diff
@@ -158,9 +163,13 @@ public void configurePathMatch(PathMatchConfigurer configurer) {
 	}
 
 	@Bean
-	@ConditionalOnMissingBean
+	@ConditionalOnMissingBean(TapOnChannelRecoveryStrategy.class)
```

Comment:
I believe it is better to have `ConditionalOnMissingBean` in all the beans defined in this admin configuration to provide an ability to extend/override the beans from this `starter` project to the project that includes it.